src/router.ts

```diff
@@ -1140,6 +1141,31 @@ export async function handleSlackVoterThreadMessage(
     slackRetryNum: retryCount,
     slackRetryReason: retryReason,
   });
+  outboundDbMessageEntry.slackFiles = reqBody.event.files;
+
+  // check attachments
+  if (outboundDbMessageEntry.slackFiles) {
```

Comment:
The `outboundDbMessageEntry` shouldn't be responsible for carrying data between functions except for the purpose of logging to the database. Can you pass `slackFiles` via the `options` object instead?